Library/Homebrew/messages.rb

```diff
@@ -0,0 +1,36 @@
+# A Messages object collects messages that may need to be displayed together
+# at the end of a multi-step `brew` command run
+class Messages
+  attr_reader :caveats, :formula_count
+
+  def initialize
+    @caveats = []
+    @formula_count = 0
+  end
+
+  def record_caveats(f, caveats)
+    @caveats.push([f, caveats])
+  end
+
+  def formula_installed(_f)
+    @formula_count += 1
+  end
+
+  def display_messages
+    display_caveats
+  end
+
+  def display_caveats
+    return unless @formula_count > 1
```

Comment:
`return if @formula_count < 1`

Reply:
Amended.

Reply:
Repository lurker sharp-sighting:  Shouldn't that be '`<=`,' not just '`<`' (to properly be the true reverse of '`>`?')

Reply:
Yep. The amended commit uses `return if @formula_count <= 1`.